requirements/constraints.txt

```diff
@@ -15,7 +15,9 @@
 # the next major release, ensure the installed version is within boundaries.
 celery>=5.2.2,<6.0.0
 
-django-simple-history==3.4.0
+# click and packaging are pinned to avoid upgrade issues
```

Comment:
Can you link to an unpinning ticket that provides more details? Could we alphabetize and duplicate comment, only no need to mention both. Maybe they each get their own unpinning ticket.

Reply:
Linked the unpin ticket with constraint.